.github/workflows/PR.yml

```diff
@@ -52,6 +52,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/docs
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
iirc secrets starting with "GITHUB_" are not supported

Reply:
> iirc secrets starting with "GITHUB_" are not supported


what do you mean ?

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

and it works in fc33a96 as well